src/components/ProductList/cart.js

```diff
@@ -13,8 +15,15 @@ function ProductListCart() {
 			{cartProducts.map(cartProduct => {
 				return (
 					<StyledProductCard key={cartProduct.id}>
-						<p>{cartProduct.name}</p>
-						<p>{cartProduct.price}</p>
+						<h1>{cartProduct.name}</h1>
+						<h2>{cartProduct.price}</h2>
+						<Image
+							alt={'Photo from the product'}
+							src={cartProduct.imgUrl}
+							width={100}
+							height={100}
+							layout="fixed"
+						/>
```

Comment:
🔔 **Ring the the redundant code alarm**  =>  ProductList/cart.js and ProductList/index, both rendering a Card. 
Think about outsourcing this as a reusable component.